app.py

```diff
@@ -70,12 +71,45 @@ def main() -> None:
     )
 
     # tag all resources in stack
+    git_branch = (
+        subprocess.Popen(
+            ["git", "rev-parse", "--abbrev-ref", "HEAD"], shell=False, stdout=subprocess.PIPE
+        )
+        .communicate()[0]
+        .decode()
+        .strip()
+    )
+    git_commit = (
+        subprocess.Popen(
+            ["git", "rev-parse", "--short", "HEAD"], shell=False, stdout=subprocess.PIPE
+        )
+        .communicate()[0]
+        .decode()
+        .strip()
+    )
+
+    git_tag = (
+        subprocess.Popen(
+            ["git", "describe", "--tags", "--exact-match"],
+            shell=False,
+            stdout=subprocess.PIPE,
+            stderr=subprocess.DEVNULL,
+        )
+        .communicate()[0]
+        .decode()
+        .strip()
+    )
+    if not git_tag:
+        git_tag = "UNRELEASED"
+
     Tag.add(app, "CostCentre", "100005")
     Tag.add(app, APPLICATION_NAME_TAG_NAME, APPLICATION_NAME)
     Tag.add(app, "Owner", "Bill M. Nelson")
     Tag.add(app, "EnvironmentType", ENV)
     Tag.add(app, "SupportType", "Dev")
     Tag.add(app, "HoursOfOperation", "24x7")
+    Tag.add(app, "GitVersion", f"{git_branch}/{git_commit}")
```

Comment:
Could we split this up into branch + commit separately? Just in case we ever need to use the value in code, so we don't need to parse it.

Reply:
Git information split in to `GitBranch` and `GitVersion` tags.